packages/server/api/routes/retro.router.js

```diff
@@ -1,13 +1,66 @@
 const express = require('express');
 
 const router = express.Router({ mergeParams: true });
+
+// controllers
 const retroController = require('../controllers/retroController');
 
+router.get('/', (req, res, next) => {
+  retroController
+    .getRetro()
+    .then((result) => res.json(result))
+    .catch(next);
+});
+
+router.get('/:id', (req, res, next) => {
+  retroController
+    .getRetroById(req.params.id)
+    .then((result) => res.json(result))
+    .catch(next);
+});
+
+router.post('/', (req, res) => {
+  retroController
+    .createRetro(req.body)
+    .then((result) => res.json(result))
+    .catch((error) => {
+      // eslint-disable-next-line no-console
+      console.log(error);
+      res.status(400).send('Bad request').end();
+    });
+});
+
+router.post('/add', (req, res) => {
+  retroController
+    .addRetro(req.body)
+    .then((result) => res.json(result))
+    .catch((error) => {
+      // eslint-disable-next-line no-console
+      console.log(error);
+      res.status(500).send('Failed to add retro').end();
+    });
+});
+
 router.post('/', (req, res, next) => {
   retroController
     .createTeam(req.body)
```

Comment:
You don't have `createTeam` in the retroController.
I recommend to remove this block of code if it's now used for anything.

Reply:
Done